ambry-cloud/src/main/java/com/github/ambry/cloud/HelixVcrCluster.java

```diff
@@ -171,22 +168,4 @@ public void close() {
     manager.disconnect();
     helixAdmin.close();
   }
-
-  /**
-   * Get the name of the state model for the factory class.
-   * @param stateModelFactoryClass name of the {@link org.apache.helix.participant.statemachine.StateModelFactory} class.
-   * @return state model name corresponding to the factor class.
-   */
-  private String getStateModelNameFromFactory(String stateModelFactoryClass) {
-    if (stateModelFactoryClass.equals(OnlineOfflineHelixVcrStateModelFactory.class.getName())) {
```

Comment:
Minor: could use equalsIgnoreCase (optional)

Reply:
This code has been removed. Each `VcrStateModelFactory` class now maintains its own statemodel name, so there is no longer any need for this comparison anymore.